libraries/ms-common/inc/gpio_it.h

```diff
@@ -4,7 +4,7 @@
 #include <stdint.h>
 
 #include "gpio.h"
-#include "interrupt.h"
+#include "interrupt_def.h"
```

Comment:
It's necessary for the interface definition since this is just the architecture universal header we don't include the architecture specific interrupt headers.
`StatusCode gpio_it_register_interrupt(GPIOAddress *address, InterruptSettings *settings,
                                      InterruptEdge edge, gpio_it_callback callback, void *context);`